tests/contrib/langchain/test_langchain.py

```diff
@@ -1256,3 +1258,90 @@ def test_vectorstore_logs_error(langchain, ddtrace_config_langchain, mock_logs,
             "documents": [],
         }
     )
+
+
+@pytest.mark.parametrize("ddtrace_global_config", [dict(_llmobs_enabled=True, _llmobs_sample_rate=1.0)])
```

Comment:
will update with the `_llmobs_ml_app` option once #8691 is merged, or in one of the follow up PRs for langchain span events

Reply:
Let's update here, thanks!